microcosm_pubsub/registry.py

```diff
@@ -56,7 +56,8 @@ def register(self, media_type, value):
 
     def find(self, media_type):
         """
-        Create a codec or raise KeyError.
+        Create a codec or raise KeyError. If autoregistration is enabled, falls
```

Comment:
Not sure if this is good enough, but this was the magic that got me stuck for a while. In practice most of our code uses autoregistration to implicitly treat things with the URIMessageSchema so we don't have many examples of explicitly wiring messages into the schema in our code.